image/image.go

```diff
@@ -202,6 +202,18 @@ func bootstrapToRootDir(sto Store, model *asserts.Model, opts *Options) error {
 			return err
 		}
 
+		// fetch the snap assertions too
+		sha3_384, _, err := asserts.SnapFileSHA3_384(fn)
+		ref := &asserts.Ref{
+			Type:       asserts.SnapRevisionType,
+			PrimaryKey: []string{sha3_384},
+		}
+		if err := f.Fetch(ref); err != nil {
+			logger.Noticef("cannot fetch assertion %q: %s", ref.Unique(), err)
```

Comment:
I don't understand why just logging here? we need again an envvar or some option to choose this, an image will not actually boot without those assertions